### ISSUE TITLE
O2 build fails at installation in Utilities/QC/QCMetricsExtractor

### ISSUE THREAD

**Issue by matthiasrichter** [AliceO2Group/AliceO2#196]
@Barthelemy, @PatrykLesiak compiling dev branch at 6f48149e4e3f44e3fe13b794432f17d2146ba01d
fails at installation

```bash
CMake Error at Utilities/QC/QCMetricsExtractor/cmake_install.cmake:56 (file):
  file INSTALL cannot find
  "/src/src/alisw/sw/SOURCES/O2/dev/0/Utilities/QC/QCMetricsExtractor/include/QCMetricsExtractor".
Call Stack (most recent call first):
  Utilities/QC/cmake_install.cmake:41 (include)
  Utilities/cmake_install.cmake:38 (include)
  cmake_install.cmake:51 (include)

**Comment by Barthelemy** [comment]
Hi, 
Ok, I will have a look.

**Comment by Barthelemy** [comment]
With the default o2-daq it compiles. I will now try with the default "o2".

**Comment by Barthelemy** [comment]
On which platform do you get this ? because I can't reproduce it on CC7.

**Comment by matthiasrichter** [comment]
@Barthelemy its ubuntu 14.04
I also had a quick look in the CMakeList.txt, but I could not find anything obvious. It's strange that it tries to install a file from the SOURCE with the name of the executable

**Comment by PatrykLesiak** [comment]
Dear @matthiasrichter,

Could you please provide us with commands which you are using to build project ? Are you using "make install" ? Are you building the whole O2 project or just QC module ?

**Comment by Barthelemy** [comment]
Actually I am hitting another problem : DDS is not detected and thus QC is not built. This might explain why I don't see your problem @matthiasrichter . Did you set any particular variable to get DDS detected by O2 ?

**Comment by matthiasrichter** [comment]
Hi Patryk, Barth,
yes, I also hit the DDS pick-up-problem before, that should be fixed in the latest alidist, see
https://github.com/AliceO2Group/AliceO2/issues/186

I'm doing an installation from scratch in a clean directory, here are the commands:
```bash
git clone https://github.com/alisw/alibuild
git clone https://github.com/alisw/alidist
alibuild/aliBuild --defaults o2 --disable Python-modules build o2
```

The ```--disable Python-modules``` is just a workaround for https://github.com/alisw/alibuild/issues/396

Cheers
Matthias

**Comment by Barthelemy** [comment]
Ok, I can reproduce the problem.

**Comment by Barthelemy** [comment]
Missing subdirectory : https://github.com/AliceO2Group/AliceO2/pull/198

**Comment by matthiasrichter** [comment]
Fixed in PR #198
